hydrolib/tools/ext_old_to_new/main_converter.py

```diff
@@ -65,7 +54,7 @@ def ext_old_to_new(
     structurefile: PathOrStr = None,
     backup: bool = False,
     postfix: str = "",
-) -> Tuple[ExtOldModel, ExtModel, IniFieldModel, StructureModel]:
+) -> Union[Tuple[ExtOldModel, FileModel, FileModel, FileModel], None]:
```

Comment:
Why changed to these less-informative ancestor classes? The result is known to be a tuple of the specific models (Ext/Inifield, etc.)

Reply:
well as all the models as subclasses from the INIModel-> ParsableFileModel -> FileModel
so the super class is `FileModel` and the type checker basically point that all outputs are `FileModel` but i can make like what you are saying and ignore the type checker

Reply:
As discussed: let's continue to type them with the most specific known typo instead of some ancestor class.